dap_aria_mapping/pipeline/app_tables/emergence_table.py

```diff
@@ -0,0 +1,74 @@
+from dap_aria_mapping.getters.openalex import get_openalex_topics, get_openalex_works
+from dap_aria_mapping.getters.patents import get_patent_topics, get_patents
+from dap_aria_mapping.utils.app_data_utils import count_documents, expand_topic_col
+import polars as pl
+import pandas as pd
+import boto3
+import io
+#from nesta_ds_utils.loading_saving.S3 import upload_obj
+from dap_aria_mapping import BUCKET_NAME, logger
+
+if __name__ == "__main__":
+
+    logger.info("Loading patent data")
+    patents_with_topics = get_patent_topics(tax = "cooccur", level = 3)
+
+    logger.info("Transforming patent dictionary to polars df")
+    patents_with_topics_df = pl.DataFrame(
+        pd.DataFrame.from_dict(patents_with_topics, orient='index'
+        ).T.unstack().dropna().reset_index(drop=True, level=1).to_frame().reset_index())
+    patents_with_topics_df.columns = ["id", "topic"]
+
+    logger.info("Loading patent date data")
+    patent_dates = pl.DataFrame(get_patents()).select(
+        ["publication_number", "publication_date"]
```

Comment:
FYI I used `"priority_date"` when calculating novelty of patents - perhaps OK to keep it like that for novelty? (it's sort of a bit closer the year when the "ideas" (ie, topic combinations) were created). Probably doesn't matter much at this point for the dashboard anyways - but if you wanna go with publication date, I can also create an issue to change that later this week.

Reply:
Hmm - maybe we should have this conversation on standup?